lib/src/components/dial_pad/dial_pad.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+import '../../../zeta_flutter.dart';
+
+const Map<String, String> _defaultButtonValues = {
+  '1': '',
+  '2': 'ABC',
+  '3': 'DEF',
+  '4': 'GHI',
+  '5': 'JKL',
+  '6': 'MNO',
+  '7': 'PQRS',
+  '8': 'TUV',
+  '9': 'WXYZ',
+  '*': '',
+  '0': '+',
+  '#': '',
+};
+
+const int _defaultButtonsPerRow = 3;
+
+/// Dial pad gives the user the ability to dial a number and start a call. It also has a quick dial security action and a delete entry action.
+class ZetaDialPad extends StatelessWidget {
+  /// Constructs a [ZetaDialPad].
+  const ZetaDialPad({
+    super.key,
+    this.onInput,
+    this.buttonsPerRow = _defaultButtonsPerRow,
+    this.buttonValues = _defaultButtonValues,
+  });
+
+  /// Callback when number is tapped. Returns the large value from the button, i,e, 1,2,3 etc.
+  final ValueChanged<String>? onInput;
```

Comment:
I couldnt think of a good way to do it, I guess I could add in a function that returns the chars with some delay?

Reply:
it's been so long since I've used one of those keypads I can't remember how they work. Is the first tap the numbers and then subsequent taps cycle through the letters?

Reply:
I think it either works in letter or number mode. Otherwise entering 999 would go to Y